styleguide/source/assets/css/scss/generic/_mixins.scss

```diff
@@ -26,6 +26,11 @@
   border-top: $width $color $style;
 }
 
+// Horizontal rule
+@mixin rule-horizontal-bottom($width, $color, $style) {
```

Comment:
We have an atom for rule-horizonal. Is there a reason for preferring to use a mixin over `{% include' atoms-rule-horizontal' %}` in this situation? I feel like the latter has been our preference up until now, so I'm just curious if we think that's a practice that we should stop doing.

I know the mixin is how we do things in D7, but since this is D8...